integration-tests/terraform/eks/main.tf

```diff
@@ -122,6 +122,6 @@ resource "null_resource" "validator" {
     aws_eks_addon.this
   ]
   provisioner "local-exec" {
-    command = "go test ${var.test_dir} -v"
+    command = "go test ${var.test_dir} -v --tags=linuxonly"
```

Comment:
will this prevent the windows tests to run? what if we have an eks cluster on windows?